frontend/src/metabase/query_builder/components/view/sidebars/ModelCacheManagementSection/ModelCacheManagementSection.tsx

```diff
@@ -59,6 +59,11 @@ export function ModelCacheManagementSection({ model }: Props) {
   const isError = persistedModel?.state === "error";
   const lastRefreshTime = moment(persistedModel?.refresh_end).fromNow();
 
+  const canRefreshCache =
+    persistedModel && checkCanRefreshModelCache(persistedModel);
+  const refreshTooltip =
+    persistedModel?.state === "creating" ? t`Create now` : t`Refresh now`;
```

Comment:
*Thinking out loud. This isn't a suggestion.*

If you move this ternary to line 91, in place of `refreshTooltip`, you can avoid using `?.` which might simplify the code in one respect. But that means moving the ternary into the JSX, which complicates the code in another respect.